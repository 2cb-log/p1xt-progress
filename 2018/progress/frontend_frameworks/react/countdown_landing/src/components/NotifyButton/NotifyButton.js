import React from 'react';
import './NotifyButton.css';

function NotifyButton(props) {
  return (
    <div className="NotifyButton">
      NotifyButton
    </div>
  );

}

export default NotifyButton;
