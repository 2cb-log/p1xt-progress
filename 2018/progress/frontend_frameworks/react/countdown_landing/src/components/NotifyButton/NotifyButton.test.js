import React from 'react';
import ReactDOM from 'react-dom';
import NotifyButton from './NotifyButton';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<NotifyButton />, div);
  ReactDOM.unmountComponentAtNode(div);
});