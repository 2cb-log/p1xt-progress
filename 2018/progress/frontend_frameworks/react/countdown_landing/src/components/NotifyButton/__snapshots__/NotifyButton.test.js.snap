// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotifyButton /> renders correctly 1`] = `
<div
  className="NotifyButton"
>
  NotifyButton
</div>
`;
